doc/high_availability.md

```diff
@@ -1,110 +1,74 @@
 High Availability (HA)
 ======================
```

Comment:
Based on the content that follows, the topic here is just _controller_ high availability, so I'd update the title to that.